packages/webapp/public/index.html

```diff
@@ -4,6 +4,7 @@
     <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet" />
```

Comment:
Remove this file since it is only for testing.

Reply:
Done